Topic tuneup - sales folder, pt 2 - GH 725 (#870)
magento/merchdocs: src/sales/invoice-create.md

|Capture Offline |When the invoice is submitted, the system does not capture the payment. It is assumed that the payment is captured directly through the gateway, and you no longer have the option to capture this payment through Magento. You have the ability to create a credit memo, but you do not have the option to void the invoice. (Even though the order used an online payment, the invoice is essentially an offline invoice.)
|Not Capture |When the invoice is submitted, the system does not capture the payment. It is assumed that you will capture the payment through Magento at a later date. There is a _Capture_ button in the completed invoice. Before capturing, you are able to cancel the invoice. After capturing you are able to create a credit memo and void the invoice.

{:.bs-callout-info}
Do not select the _Not Capture_ option unless you are certain that you are going to capture the payment through Magento at a later date. You cannot create a credit memo until the payment has been captured using the Capture button.

## Invoice an order

> You must generate an invoice for an order before you can [print it]. To view or print the PDF, first download and install a PDF reader such as [Adobe Acrobat Reader][1].

Author: ```suggestion
You must generate an invoice for an order before you can print it. To view or print the PDF, you must have a PDF reader. You can download [Adobe Reader][1] at no charge.
```
